source/_integrations/launch_library.markdown

```diff
@@ -1,7 +1,6 @@
 ---
 title: Launch Library
 description: Instructions on how to integrate space launch information within Home Assistant.
-logo: rocket.png
```

Comment:
This file is not used by anything else and should be deleted in this Pr as well `source/images/supported_brands/rocket.png`